### DISCUSSION TITLE
Granting Database roles to other roles and access privileges to Database Roles ob objects

### DISCUSSION THREAD

**Discussion by cpmub1** [Snowflake-Labs/terraform-provider-snowflake#2024] [Q&A]
Hi, i see the new resource type - database role for creation. But, i dont see any resources for granting the database role to other roles or granting access privileges to database roles.

can someone help?

**Comment by sfc-gh-asawicki** [comment]
Hey @cpmub1. Thanks for the question!

Yes, this is true, there are yet no resources to utilize the grant functionalities for database roles.

We are now forming a bigger team to work on the project, and we see the need to implement resources to handle grants, not only for the database role but also for other objects. That's why we have bumped it on our priorities list so that we will hopefully handle it in October/November.
